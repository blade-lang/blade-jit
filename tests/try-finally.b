try {
  var a = 20
  raise new Error('Something happened')
} finally {
  echo 'Cleaning up'
}