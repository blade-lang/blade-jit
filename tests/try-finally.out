^^^
Cleaning up
Error: Something happened
	at <nim> @.script(tests/try-finally.b:3:21-58)